src/vs/workbench/contrib/notebook/common/model/notebookCellTextModel.ts

```diff
@@ -113,12 +112,7 @@ export class NotebookCellTextModel extends Disposable implements ICell {
 			return this._textBuffer;
 		}
 
-		const builder = new PieceTreeTextBufferBuilder();
-		builder.acceptChunk(this._source);
-		const bufferFactory = builder.finish(true);
-		const { textBuffer, disposable } = bufferFactory.create(model.DefaultEndOfLine.LF);
-		this._textBuffer = textBuffer;
-		this._register(disposable);
```

Comment:
@rebornix Replacing this as I need to create text buffers for metadata as well, and figure i might as well change this here too.